### PR TITLE
SDK-5463: Adjust the upgrade process so it runs phpstan between itera…

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/SDK-5463

### CODE REVIEW

File: config/DynamicEvaluator/checker_phpstan_include_project.neon [hunk 1]
```diff
@@ -1,5 +1,3 @@
-includes:
```
Comment (vol4onok) [inline config/DynamicEvaluator/checker_phpstan_include_project.neon:1]: Revert functionality related to the project settings for php stan

Comment (VladislavStrelchenko) [reply config/DynamicEvaluator/checker_phpstan_include_project.neon:1]: ++

**Review by vol4onok — APPROVED** [review]

File: config/DynamicEvaluator/checker_phpstan.neon [hunk 2]
```diff
@@ -6,3 +6,5 @@ parameters:
         analyse:
             - %currentWorkingDirectory%/src/Generated/*
             - %currentWorkingDirectory%/src/Orm/*
+            - %currentWorkingDirectory%/src/Pyz/Zed/*/Persistence/*Repository.php
```
Comment (vol4onok) [inline config/DynamicEvaluator/checker_phpstan.neon:9]: maybe better to ignore them like this             `- %currentWorkingDirectory%/src/*QueryContainer.php`
all files that ends *Repository.php & *QueryContainer.php . for now it's specific case for concrete project

Comment (vol4onok) [reply config/DynamicEvaluator/checker_phpstan.neon:9]: btw why have you ignored them?

Comment (VladislavStrelchenko) [reply config/DynamicEvaluator/checker_phpstan.neon:9]: It is part of Spryker Php Syan configuration

**Pull request merged**